java/piranha/config/properties.json

```diff
@@ -0,0 +1,46 @@
+{
+  "piranhaMethodProperties": [
+    {
+      "methodName": "isToggleDisabled",
+      "flagType": "control"
+    },
+    {
+      "methodName": "isFlagTreated",
+      "flagType": "treated",
+      "returnType": "boolean"
+    },
+    {
+      "methodName": "isToggleEnabled",
+      "flagType": "treated",
+      "returnType": "boolean",
+      "receiverType": "com.uber.piranha.*.XPTest"
+    },
+    {
+      "methodName": "putToggleEnabled",
+      "flagType": "empty"
+    },
+    {
+      "methodName": "putToggleDisabled",
+      "flagType": "empty"
+    },
+    {
+      "methodName": "includeEvent",
+      "flagType": "empty"
+    },
+    {
+      "methodName": "isToggleInGroup",
+      "flagType": "treatmentGroup"
+    },
+    {
+      "methodName": null,
```

Comment:
don't follow the need for the set of `null` here.

Reply:
The last set is just for illustrative purposes. Do you think it's better to replace `null` with empty strings `""` or just remove that set altogether?

Reply:
I suggest that it be changed to `...` from line 34, as there is no requirement to have a `null` or empty string at the end of the array.

Reply:
:+1: changing to `...` in README.md and deleting in the actual `properties.json` as it has to be JSON formatted.